client/src/components/History/CurrentHistory/HistoryFilters/filterConversion.js

```diff
@@ -34,7 +34,7 @@ export function getFilterText(filterSettings) {
     let newFilterText = "";
     Object.entries(filterSettings).forEach(([key, value]) => {
         const skipDefault = hasDefaults && normalized[key] !== undefined;
-        if (!skipDefault && value !== undefined) {
+        if (!skipDefault && ((normalized[key] !== undefined && value !== undefined) || value)) {
```

Comment:
What is the purpose of `(normalized[key] !== undefined && value !== undefined)` here?

Reply:
So, if I just did `(!skipDefault && value)`, if a default filter was changed, that would not be considered, thats why:
`(normalized[key] !== undefined && value !== undefined)` means: if it is a default filter and isn't undefined